net/databag/files/databag.init

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh /etc/rc.common
+ 
+START=10
+STOP=15
+
+config_databag() {
+	config_get store_path "$1" store_path "/tmp/store"
+	config_get service_port "$1" service_port "7001"
+
+	mkdir -p "${store_path}"
+	cp -n /opt/databag/databag.db ${store_path}
```

Comment:
shouldn't database be persisted? `ln -sf` might be a better approach

Reply:
The .db will only persist changes when a writable device is mounted and the store_path is updated to that location. The initial .db file will copy the first time, following attempts will not overwrite leaving a persisted copy. Perhaps I don't understand, but I think a symlink to the installed location won't persist changes.

Reply:
okay... i see it